src/settings.js

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const _ = require('lodash');
+const { config: hathorLibConfig, walletUtils, errors: hathorLibErrors } = require('@hathor/wallet-lib');
+
+const errors = require('./errors');
+const { initializedWallets } = require('./services/wallets.service');
+const { removeAllWalletProposals } = require('./services/atomic-swap.service');
+const { initHathorLib } = require('./app');
+
+/**
+ * The multisig configuration object
+ * @typedef {Object} MultiSigConfig
+ * @property {number} total - Total number of participants, must be equal to the length of `pubkeys`.
+ * @property {number} numSignatures - Number of signatures required to send a transaction, must be lower than `total`.
+ * @property {string[]} pubkeys - list of participant xpubkeys.
+ */
+
+/**
+ * @typedef {Object} PluginConfig
+ * @property {string} name Plugin name
+ * @property {string} file Plugin filename inside the plugin directory.
+ */
+
+/**
+ * The configuration object
```

Comment:
Maybe we should have this type properly defined elsewhere but since we do not use typescript in this project I went with JSDoc, to help keep track of all variations of the config we should check.